### PR TITLE
Add Plugin: lektor-markdown-ruby-blocks

### PR DESCRIPTION
**Pull request by 25349023**
- [GitHub Page](https://github.com/25349023/lektor-markdown-ruby-blocks)
- [PyPI Page](https://pypi.org/project/lektor-markdown-ruby-blocks/)

### CODE REVIEW

File: content/plugins/lektor-markdown-ruby-blocks/lektor-markdown-highlighter/contents.lr [hunk 1]
```diff
@@ -0,0 +1,13 @@
+_model: plugin
```
Comment (dairiki) [inline content/plugins/lektor-markdown-ruby-blocks/lektor-markdown-highlighter/contents.lr:1]: This file is in the wrong location.

It needs to be in `content/plugins/lektor-markdown-ruby-blocks/contents.lr`.

Right now it's one level too deep in `content/plugins/lektor-markdown-ruby-blocks/lektor-markdown-highlighter/contents.lr`. As such, Lektor does not find and build the page.

**Review by dairiki — COMMENTED** [review]
Thank you for the submission.

However, as is, Lektor is not building the plugin page for your plugin, cause you have the `contents.lr` file one directory too deep.

**Comment by 25349023** [comment]
Thank you for pointing out the mistake, I have corrected it.

**Comment by dairiki** [comment]
@25349023  It's not at all your fault, but your use of PEP621 to put your project metadata into your `pyproject.toml` is revealing issues in `lektor-website`s code.  It'll take a little while to sort those out before we merge your PR.  Standby...

**Comment by dairiki** [comment]
@25349023 Note the suggestion at https://www.getlektor.com/docs/plugins/publishing/#listing-on-this-site that your distribution _should_ probably include the `Framework :: Lektor`, `Environment :: Web Environment`, and `Environment :: Plugins` classifiers in its metadata.  (These make it easier to find your plugin on PyPI.)

**Comment by 25349023** [comment]
OK. I've updated my package to include those classifiers.

**Comment by dairiki** [comment]
Merged.  Thank you!

**Pull request merged**